plugin-server/src/main/pluginsServer.ts

```diff
@@ -335,13 +333,6 @@ export async function startPluginsServer(
             schedule.scheduleJob('*/5 * * * *', async () => {
                 await piscina?.broadcastTask({ task: 'reloadAllActions' })
             })
-            // every 5 seconds set Redis keys @posthog-plugin-server/ping and @posthog-plugin-server/version
-            schedule.scheduleJob('*/5 * * * * *', async () => {
-                await hub!.db!.redisSet('@posthog-plugin-server/ping', new Date().toISOString(), 60, {
```

Comment:
That redis key is used by [the app's preflight logic](https://github.com/PostHog/posthog/blob/6f29a9c0347ae792b2a5509a6fae5a6db3e17c22/posthog/utils.py#L782-L787). We need to keep it or change the preflight to check for another condition.

Reply:
ah damn I'd missed that 👍

Reply:
laaaaaaame

Reply:
Have added that back 😓